app/models/task.py

```diff
@@ -1,6 +1,23 @@
 from flask import current_app
 from app import db
-
+from datetime import datetime
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String(50))
+    description = db.Column(db.String(100))
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+
+    def to_json(self):
+        if self.completed_at:
+            check_completion = True
+        else:
+            check_completion = False
```

Comment:
This can be simplified as:

```suggestion
        check_completion = bool(self.completed_at)
```